custom/icds_reports/ucr/data_sources/ag_care_monthly.json

```diff
@@ -0,0 +1,715 @@
+      },
+      "filter_expression": {
+        "type": "or",
+        "filters": [
+          {
+            "operator": "eq",
+            "type": "boolean_expression",
+            "expression": {
+              "type": "root_doc",
+              "expression": {
+                "datatype": null,
+                "type": "property_name",
+                "property_name": "closed"
+              }
+            },
+            "property_value": false,
+            "comment": null
+          },
+          {
+            "operator": "gte",
+            "type": "boolean_expression",
+            "expression": {
+              "type": "diff_days",
+              "from_date_expression": {
+                "type": "month_start_date",
+                "date_expression": {
+                  "date_expression": {
+                    "type": "icds_datetime_now"
+                  },
+                  "type": "add_months",
+                  "months_expression": {
+                    "type": "evaluator",
+                    "context_variables": {
+                      "iteration": {
+                        "type": "identity"
+                      }
+                    },
+                    "statement": "iteration - 2",
+                    "datatype": null
+                  }
+                }
+              },
+              "to_date_expression": {
+                "type": "root_doc",
+                "expression": {
+                  "datatype": "date",
+                  "type": "property_name",
+                  "property_name": "closed_on"
+                }
+              }
+            },
+            "property_value": 0,
+            "comment": null
+          }
+        ]
+      }
+    },
+    "configured_filter": {
+      "operator": "eq",
+      "type": "boolean_expression",
+      "expression": {
+        "type": "property_name",
+        "property_name": "type",
+        "datatype": null
+      },
+      "property_value": "ag_care",
+      "comment": null
+    },
+    "validations": [
+      {
+        "name": "owned_by_awc",
+        "error_message": "Case not owned by AWC",
+        "expression": {
+          "type": "boolean_expression",
+          "operator": "eq",
+          "expression": {
+            "location_id_expression": {
+              "type": "property_name",
+              "property_name": "owner_id"
+            },
+            "type": "location_type_name"
+          },
+          "property_value": "awc"
+        }
+      }
+    ],
+    "configured_indicators": [
+      {
+        "display_name": null,
+        "datatype": "string",
+        "expression": {
+          "expression": {
+            "datatype": null,
+            "type": "property_name",
+            "property_name": "owner_id"
+          },
+          "type": "root_doc"
+        },
+        "is_primary_key": false,
+        "transform": {},
+        "is_nullable": false,
+        "type": "expression",
+        "column_id": "awc_id",
```

Comment:
this is the same as `owner_id`

Reply:
Removed this column in 3bd57918710, fyi Akshita/Sunaina

Reply:
offlined, reverted 3bd57918710